### PR TITLE
[OCPBUGS3-0079] Change url and variables from {} to <>

### PR DESCRIPTION
**Pull request by SNiemann15**
Version(s): 4.14 +

Issue: https://issues.redhat.com/browse/OCPBUGS-30079

Link to docs preview: https://72446--ocpdocs-pr.netlify.app/openshift-enterprise/latest/installing/installing_sno/install-sno-installing-sno#install-sno-with-ibmz

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/install-sno-ibm-z-kvm.adoc [hunk 1]
```diff
@@ -149,21 +149,21 @@ The file names contain the {product-title} version number. They resemble the fol
...
-   --extra-args "nameserver=${NAME_SERVER}" \
+   --extra-args "ip=<ip>::<gateway>:<mask>:<hostname>::none" \
+   --extra-args "nameserver=<name_server>" \
    --extra-args "ip=dhcp rd.neednet=1 ignition.platform.id=metal ignition.firstboot" \
-   --extra-args "coreos.live.rootfs_url={rhcos_liveos}" \// <2>
-   --extra-args "ignition.config.url={rhcos_ign}" \// <3>
+   --extra-args "coreos.live.rootfs_url=<rhcos_liveos>" \// <2>
+   --extra-args "ignition.config.url=<rhcos_ign>" \// <3>
    --extra-args "random.trust_cpu=on rd.luks.options=discard" \
    --extra-args "console=tty1 console=ttyS1,115200n8" \
```
Comment (smolin-de) [inline modules/install-sno-ibm-z-kvm.adoc:168]: Use "console=ttysclp0" \
instead: "console=tty1 console=ttyS1,115200n8" \

**Review by skopacz1 — APPROVED** [review]
LGTM!

File: modules/install-sno-ibm-z.adoc [hunk 2]
```diff
@@ -33,7 +33,7 @@ $ ARCH=<architecture> <1>
 +
 [source,terminal]
 ----
-$ curl -k https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$OCP_VERSION/openshift-client-linux.tar.gz -o oc.tar.gz
+$ curl -k https://mirror.openshift.com/pub/openshift-v4/s390x/clients/ocp/<ocp_version>/openshift-client-linux.tar.gz -o oc.tar.gz
```
Comment (smolin-rh) [inline modules/install-sno-ibm-z.adoc:36]:
```suggestion
$ curl -k https://mirror.openshift.com/pub/openshift-v4/${ARCH}/clients/ocp/${OCP_VERSION}/openshift-client-linux.tar.gz -o oc.tar.gz
```

File: modules/install-sno-ibm-z.adoc [hunk 3]
```diff
@@ -50,7 +50,7 @@ $ chmod +x oc
 +
 [source,terminal]
 ----
-$ curl -k https://mirror.openshift.com/pub/openshift-v4/clients/ocp/$OCP_VERSION/openshift-install-linux.tar.gz -o openshift-install-linux.tar.gz
+$ curl -k https://mirror.openshift.com/pub/openshift-v4/s390x/clients/ocp/<ocp_version>/openshift-install-linux.tar.gz -o openshift-install-linux.tar.gz
```
Comment (smolin-rh) [inline modules/install-sno-ibm-z.adoc:53]:
```suggestion
$ curl -k https://mirror.openshift.com/pub/openshift-v4/${ARCH}/clients/ocp/${OCP_VERSION}/openshift-install-linux.tar.gz -o openshift-install-linux.tar.gz
```

**Review by smolin-rh — COMMENTED** [review]
See my suggestions/improvements.

File: modules/install-sno-ibm-z.adoc (hunk 2 above)
Comment (SNiemann15) [reply modules/install-sno-ibm-z.adoc]: Discussed with @smolin-rh that the reason to not use angle brackets in this case is that you can copy the command and run without further changes. Other option would have been to remove step 1 and 2.

Comment (skopacz1) [reply modules/install-sno-ibm-z.adoc]: New changes LGTM

**Comment by bscott-rh** [comment]
Merge review LGTM.

**Pull request merged**